assets/src/realtime.d.ts

```diff
@@ -65,6 +65,7 @@ export type NotificationReason =
   | "adjusted"
   | "operator_error"
   | "traffic"
+  | string
```

Comment:
@skyqrose I'm not clear on why we're adding support for arbitrary reason strings: we control both ends of the pipe, so we can ensure that only recognized reasons are sent.